impl/client/src/main/javascript/web/util/_a11y.js

```diff
@@ -35,6 +35,415 @@ define(["./_focus"], function(focusUtil) {
     delete: 46
   };
 
+  function makeAccessibleSelect(elem) {
+    var listboxNode = elem;
```

Comment:
Why a separate variable? Does the value of either `listboxNode` or `elem` ever change?